docs/theming.md

```diff
@@ -30,4 +30,27 @@ body {
 
 Then, set the `FRONTEND_CSS_URL` environment variable to the URL of your custom CSS file. Once you've done this, our application will load your custom CSS file and apply the styles, changing the background color to the custom color you specified.
 
+----
+
+# **Footer Configuration** 📝
+
+The footer is configurable from the theme customization file.
+
+### Settings 🔧
+
+```shellscript
+THEME_CUSTOMIZATION_FILE_PATH=<path>
+```
+
+### Example of JSON
+
+The json must follow some rules: https://github.com/suitenumerique/docs/blob/main/src/backend/impress/configuration/theme/default.json
```

Comment:
To illustrate the rules, can we point it to the demo.json in helm directory instead of the default.json?

Some customizations, like the translations do not make sense to be in default.json (yet).
Instead i've put them in demo.json to illustrate how it is used. 
If the user views this documentation, maybe it is better to link to the same files

Reply:
Yes make sense.